scripts/check_code_style.py

```diff
@@ -28,6 +28,28 @@
 import sys
 import argparse
 import os
+from pathlib import Path
+
+def sanitize_file_list(file_list: list[str]):
+    """
+    Remove files from the list of skip folders which are folders
+    we do not want to execute clang-format analysis of the files.
+    """
+    skip_folders = [
```

Comment:
Make this module-level and pass it in from check_code_style

Reply:
Using only CMake now.